modules/simple_forms_api/app/services/simple_forms_api/intent_to_file.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_claims/service'
+
+module SimpleFormsApi
+  class IntentToFile
+    attr_reader :params, :icn
+
+    def initialize(params, icn)
+      @params = params
+      @icn = icn
+    end
+
+    def submit
+      types = params['benefit_selection'].split(',')
+      ssn = if params['preparer_identification'] == 'VETERAN'
+              params.dig('preparer_id',
+                         'ssn')
+            else
+              params.dig('veteran_id', 'ssn')
+            end
+      expiration_date = ''
+      types.each do |type|
+        if existing_intents[type.downcase]
+          expiration_date = existing_intents[type.downcase]['expirationDate']
+        else
+          response = benefits_claims_lighthouse_service.create_intent_to_file(type.downcase, ssn)
+          # This only retains the last response. Is that ok?
+          # Are we safe to assume that the expiration dates for two simultaneously-created ITFs will be the same?
+          expiration_date = response.dig('data', 'attributes', 'expirationDate')
```

Comment:
@ndsprinkle, do you have any thoughts on these questions?

Reply:
@rmtolmach Not really, I assumed this would work like we expect. The response from creating a new intent to file would contain the matching expiration date, no?

Reply:
I don't really know much about Lighthouse, either. Getting the latest intent to file for a veteran seems like what you want. You'd want the latest, right? I took a peek at the [Lighthouse docs](https://developer.va.gov/explore/api/benefits-claims/docs?version=current) for `intent-to-file/{type}` and this is the example response: 
```json
{
  "data": {
    "id": "193685",
    "type": "intent_to_file",
    "attributes": {
      "creationDate": "2021-03-16T19:15:21.000-05:00",
      "expirationDate": "2022-03-16T19:15:20.000-05:00",
      "type": "compensation",
      "status": "active"
    }
  }
}
```

I'm out of my depth in this `simple_forms_api` module 😅 so I trust your judgement.

Reply:
Yep, the question here is whether some intents can have a different time window for expiration date than others. So if, for example, two intents are being created at the same time, we'll only return the expiration date for the one which happened to be created last. And the FE associated with this will present that expiration date as being applicable to both intents. But, like I said on Slack, I think it's an extreme edge case and not necessarily worth sorting through atm.